test/integration/serverless/serverless.yml

```diff
@@ -30,6 +30,18 @@ provider:
     DD_CAPTURE_LAMBDA_PAYLOAD: true
     DD_COLD_START_TRACING: false
     DD_LOG_LEVEL: DEBUG
+    DD_LOGS_CONFIG_PROCESSING_RULES: >-
+      [
+      {"type": "mask_sequences", "name": "masked_value", "replace_placeholder": "masked", "pattern": "__[\\w|_]*__"},
+      {"type": "exclude_at_match", "name": "exclude_extension_logs", "pattern": "\\d{4}-\\d{2}-\\d{2}\\s\\d{2}:\\d{2}:\\d{2}\\sUTC\\s\\|\\sDD_EXTENSION"},
+      {"type": "exclude_at_match", "name": "exclude_datadog_wrapper_logs", "pattern": "\\[datadog-wrapper\\]"},
+      {"type": "exclude_at_match", "name": "exclude_dotnet_tracer_logs", "pattern": "\\d{4}-\\d{2}-\\d{2}\\s\\d{2}:\\d{2}:\\d{2}:\\d{3}\\s\\[DD_TRACE_DOTNET\\]"},
+      {"type": "exclude_at_match", "name": "exclude_go_tracer_logs", "pattern": "\\d{4}/\\d{2}/\\d{2}\\s\\d{2}:\\d{2}:\\d{2}\\sDatadog\\sTracer"},
+      {"type": "exclude_at_match", "name": "exclude_go_tracer_logs_2", "pattern": "\\d{4}/\\d{2}/\\d{2}\\s\\d{2}:\\d{2}:\\d{2}\\s{\"status\":\"debug\",.*?}"},
+      {"type": "exclude_at_match", "name": "exclude_java_tracer_logs", "pattern": "\\[dd.trace\\s\\d{4}-\\d{2}-\\d{2}\\s\\d{2}:\\d{2}:\\d{2}:\\d{3}\\s\\+\\d{4}\\]"},
+      {"type": "exclude_at_match", "name": "exclude_node_tracer_logs", "pattern": "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}Z\\s[a-z0-9\\-]+\\sDEBUG\\s{\"status\":\"debug\",.*?}"},
+      {"type": "exclude_at_match", "name": "exclude_python_tracer_logs", "pattern": "\\[DEBUG\\]\\s\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}Z"}
```

Comment:
Correct me if I'm wrong, but if we're excluding a majority of the DEBUG level logs shouldn't we just remove the line above (`DD_LOG_LEVEL: DEBUG`)? Or if we're testing feature functionality we can update the tests to print something unique and look to exclude just that log.

Reply:
While troubleshooting the integration tests I've found it useful to have debug logs enabled to determine why exactly the tests are failing. However the debug logs to add a lot of noise to the logs being parsed for the integration tests (what's in between `BEGINLOG` and `ENDLOG`) so I added exclusion rules for the extension to not send those logs to the recorder extension.